tabular/src/autogluon/tabular/models/lgb/lgb_model.py

```diff
@@ -140,13 +140,27 @@ def _fit(self,
                     params['metric'] = train_loss_name
                 elif train_loss_name not in params['metric']:
                     params['metric'] = f'{params["metric"]},{train_loss_name}'
+            # early stopping callback will be added later by QuantileBooster if problem_type==QUANTILE
+            early_stopping_callback_kwargs = dict(
+                stopping_rounds=early_stopping_rounds,
+                metrics_to_use=[('valid_set', stopping_metric_name)],
+                max_diff=None,
+                start_time=start_time,
+                time_limit=time_limit,
+                ignore_dart_warning=True,
+                verbose=False,
+                manual_stop_file=False,
+                reporter=reporter,
+                train_loss_name=train_loss_name,
+            )
             callbacks += [
                 # Note: Don't use self.params_aux['max_memory_usage_ratio'] here as LightGBM handles memory per iteration optimally.  # TODO: Consider using when ratio < 1.
-                early_stopping_custom(early_stopping_rounds, metrics_to_use=[('valid_set', stopping_metric_name)], max_diff=None, start_time=start_time, time_limit=time_limit,
-                                      ignore_dart_warning=True, verbose=False, manual_stop_file=False, reporter=reporter, train_loss_name=train_loss_name),
+                early_stopping_custom(**early_stopping_callback_kwargs)
```

Comment:
Why are we adding this when problem_type is quantile? Wouldn't we need to remove it?